js/src/personal/haoking/Haoking.aboutme.tsx

```diff
@@ -0,0 +1,35 @@
+import { MantineProvider, Box, Flex, rem } from '@mantine/core'
+import { LeftAboutMe } from './LeftAboutMe.tsx'
+import { RightAboutMe } from './RightAboutMe.tsx'
+import { Navbar } from './Navbar.tsx'
+import { theme } from './theme.ts'
+
+export function HaokingAboutMe({
+  current,
+  setCurrent,
+}: {
+  current: string
+  setCurrent: any
+}) {
+  return (
+    <MantineProvider theme={theme}>
```

Comment:
You should have one MantineProvider for all of your pages. Not sure how that would work with the ReactRouter yet though. I guess it's fine to have one per page for now, but it's definitely not what Mantine intended.

Reply:
Removed mantine providers from all pages except main Haoking.page.tsx